Assets/Program.cs

```diff
@@ -1,12 +1,12 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-using System.Globalization;
 using Azure;
 using Azure.Identity;
 using Azure.ResourceManager;
 using Azure.ResourceManager.Media;
+using System;
```

Comment:
Normally our coding convention is to order `System.` usings before other usings (but I'm not too concerned either way).

Reply:
Visual Studio default code clean up tool is ordering alphabetically.

Reply:
Do whatever the public tooling is doing for VS or VS Code.